tox.ini

```diff
@@ -19,6 +20,7 @@ deps =
     django111: Django~=1.11
     django20: Django~=2.0
     django21: Django~=2.1
+    django21: Django~=2.2
```

Comment:
This appears to be a typo. I think you want `django22: Django~=2.2`

Reply:
true